From 917a76bdb741c534e05064c68c34857a03c177b4 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Fri, 8 Aug 2014 16:52:25 +0100
Subject: [PATCH 2/5] Removed GNAT Pro test.

---
 src/gpr_version.adb | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/gpr_version.adb b/src/gpr_version.adb
index 4ec66a8..64c82cc 100644
--- a/src/gpr_version.adb
+++ b/src/gpr_version.adb
@@ -66,8 +66,6 @@ package body GPR_Version is
       end loop Last_Loop;
 
       case Build_Type is
-         when Gnatpro =>
-            return "Pro " & Gpr_Version & " " & Date & Host;
          when GPL =>
             return "GPL " & Gpr_Version & " " & Date & Host;
          when FSF =>
-- 
1.8.5.5

