From 2803b4c1594d3a0cb7a0d98d800af7ff445d1d6c Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Fri, 30 Jan 2015 20:35:58 +0000
Subject: [PATCH] Added missing libs.

---
 gnatmem.gpr.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnatmem.gpr.in b/gnatmem.gpr.in
index 9790478..dcc7a5d 100644
--- a/gnatmem.gpr.in
+++ b/gnatmem.gpr.in
@@ -20,7 +20,8 @@ project Gnatmem is
    package Linker is
       for Default_Switches ("ada") use
          (BINUTILS_OBJDIR & "/bfd/libbfd.a",
-          BINUTILS_OBJDIR & "/libiberty/libiberty.a");
+          BINUTILS_OBJDIR & "/libiberty/libiberty.a",
+          "-lz", "-ldl");
    end Linker;
 
    package Builder is
@@ -36,4 +37,3 @@ project Gnatmem is
    end Ide;
 
 end Gnatmem;
-
-- 
2.0.5

